nixos/modules/system/boot/loader/systemd-boot/systemd-boot.nix

```diff
@@ -7,13 +7,21 @@ let
 
   efi = config.boot.loader.efi;
 
+  systemdUkify = pkgs.systemdMinimal.override {
```

Comment:
Could we instead use objcopy like [this](https://wiki.archlinux.org/title/Unified_kernel_image#Manually) to prevent another systemd variant from being built? #264163 aims to remove the `systemdMinimal` variant and changing the systemd derivation to set `useUkify = true` adds a little under 1GiB to the systemd closure by pulling in python.

Reply:
I'm not sure if I see how objcopy insert things like SBAT sections, did I miss something?

Reply:
I see how not pulling python and saving 1GiB of closure is a big advantages, but I don't see myself maintaining a complicated `objcopy` script for that feature. I have no issue if someone wants to do it and maintain in properly.

Reply:
I wonder if the systemd folks intend to replace the python script with a Rust or C program. That would certainly be more normal for them, and would allow us to enable ukify by default.

In the meantime, it would be extremely unfortunate to re-implement ukify. Even just the `objcopy` scripting is... kinda gross. But yea, it's also capable of significantly more, which we may want to take advantage of.

Reply:
Honestly, Goblin (the PE library) is like two PRs away (by me) to replace ukify if anyone is interested to help towards this and get this to completion. I am focusing on signing stuff because it's much harder to get right but the objcopy-like feature is pretty easy to replicate, just adding sections around until satisfied.

Reply:
Yeah I do agree maintaining a set of objcopy commands could get pretty unwieldy. I think one systemd variant that has `withUkify = true` would be better than multiple systemd variants throughout the entire nixos system, maybe it should be enabled by default?

Reply:
I don't know, only one variant means everyone, including those not using `systemd-boot`, including those not using `systemd-stub`, pay the cost.

Reply:
But ukify is small enough, this is not a big deal IMHO.